explorer/src/controllers/jobRuns.ts

```diff
@@ -5,20 +5,16 @@ import jobRunsSerializer from '../serializers/jobRunsSerializer'
 import jobRunSerializer from '../serializers/jobRunSerializer'
 import { getCustomRepository } from 'typeorm'
 import { JobRunRepository } from '../repositories/JobRunRepository'
+import { parseParams as parsePaginationParams } from '../utils/pagination'
```

Comment:
I think i'd prefer a namespace import here 
```suggestion
import & as pagination from '../utils/pagination'
```